src/components/App.tsx

```diff
@@ -3662,6 +3662,9 @@ class App extends React.Component<AppProps, AppState> {
     insertAtParentCenter?: boolean;
     container?: ExcalidrawTextContainer | null;
   }) => {
+    if (this.state.disableTextEditing) {
```

Comment:
Seems ok, but do we really want disable text editing entirely or are we annoyed only by double click on canvas ( `handleCanvasDoubleClick`)?

I'd say that annotating office areas with text is something desired.